woodpecker-go/woodpecker/interface.go

```diff
@@ -166,4 +166,19 @@ type Client interface {
 
 	// SetLogLevel sets the server's logging level
 	SetLogLevel(logLevel *LogLevel) (*LogLevel, error)
+
+	// CronList list all cron jobs of a repo
+	CronList(owner, repo string) ([]*CronJob, error)
+
+	// CronGet get a specific cron job of a repo by id
+	CronGet(owner, repo string, cronID int64) (*CronJob, error)
```

Comment:
In the store we mostly use `find` and `list` (I am not super happy with it), but we should stick to one schema.

Reply:
instead of id we might want to make "title" aka "name" unique and as identifyer